integration_tests/src/main/python/map_test.py

```diff
@@ -323,6 +323,14 @@ def test_simple_get_map_value_ansi_fail(data_gen):
             conf=ansi_enabled_conf,
```

Comment:
2023 copyrights

Reply:
Thank you for finding it. I accidentally turned the `auto-copyright` off. Have updated.